src/ngraph_rewrite_pass.cc

```diff
@@ -150,6 +151,16 @@ class NGraphVariableCapturePass : public NGraphRewritePass {
       DumpGraphs(options, idx, "captured", "Graph With Variables Captured");
     }
 
+    // Skip "Assert" if specifically asked by the user
+    if (std::getenv("NGRAPH_TF_SKIP_ASSERT") != nullptr) {
+      TF_RETURN_IF_ERROR(SkipAssert(options.graph->get()));
+      // If requested, dump unmarked graphs without asserts
+      if (DumpCapturedGraphs()) {
```

Comment:
Should we use a new check DumpSkipAssertGraphs() (We can check for DumpAllGraphs() and NGRAPH_TF_SKIP_ASSERT, May not want another flag) like the other ones?

Reply:
@shresthamalik good feedback. We need to refactor the code to take care of these. Please create a JIRA ticket and we will address these.

Reply:
@kanvi-nervana No